internal/database/actions_test.go

```diff
@@ -43,6 +44,31 @@ func (lp *locationInfo) ResolverIP() string {
 	return lp.resolverIP
 }
 
+func TestNewDatabase(t *testing.T) {
+	t.Run("with empty path", func(t *testing.T) {
+		dbpath := ""
+		db, err := New(dbpath)
+		if db != nil {
+			t.Fatal("unexpected database instance")
+		}
+		if err == nil || !errors.Is(err, errInvalidDatabasePath) {
```

Comment:
```suggestion
		if !errors.Is(err, errInvalidDatabasePath) {
```

The above check also covers the case where `err` is `nil`